spec/audited/audit_spec.rb

```diff
@@ -3,6 +3,61 @@
 describe Audited::Audit do
   let(:user) { Models::ActiveRecord::User.new name: "Testing" }
 
+  describe "audit class" do
+    context "when a custom audit class is configured" do
+      it "should be used in place of #{described_class}" do
+        class CustomAudit < Audited::Audit
+          def custom_method
+            "I'm custom!"
+          end
+        end
+
+        Audited.config { |config| config.audit_class = CustomAudit }
+
+        class TempModel < ::ActiveRecord::Base
+          self.table_name = :companies
+          audited
+        end
+
+        record = TempModel.create
+
+        audit = record.audits.first
+        expect(audit).to be_a CustomAudit
+        expect(audit.custom_method).to eq "I'm custom!"
+
+        Audited.config { |config| config.audit_class = nil }
```

Comment:
@domcleal I've moved this into an `around` block so that the audit class can be reset to its original value. Let me know if you'd prefer the `before`/`after` split and having `config.audit_class` reset `nil` as it is here.